src/MassTransit.RabbitMqTransport/IExchangeTypeDeterminer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace MassTransit.RabbitMqTransport
+{
+    public interface IExchangeTypeDeterminer
```

Comment:
We have a number of providers, (e.g. https://github.com/MassTransit/MassTransit/blob/develop/src/MassTransit/Hosting/IConnectionStringProvider.cs) that I think is closer naming than determiner. The purpose of the interface is to provide the exchange type. 

Worth considering are we really providing the type here, or the exchange name.